src/basic/cap-list.c

```diff
@@ -36,7 +37,7 @@ int capability_from_name(const char *name) {
         /* Try to parse numeric capability */
         r = safe_atoi(name, &i);
         if (r >= 0) {
-                if (i >= 0 && (size_t) i < ELEMENTSOF(capability_names))
+                if (i >= 0 &&)
                         return i;
```

Comment:
hmm, given that we store caps masks in a 64 bit unsigned integer we should refuse caps > 63 here